### PR TITLE
Fixes #234 - seperate messages datewise

### LINKED ISSUE #234: Separate messages datewise
**Actual Behaviour**

Currently, the message history is not separated datewise like in other clients

**Expected Behaviour**

The message date sent, can be shown to separate the messages according to the date sent.

### PR DESCRIPTION
**Pull request by uday96**
Fixes issue #234 

**Changes:**
* The messages are now seperated datewise
* Date is displayed when the message dates are different, thus seperating the messages datewise

**Demo Link:** http://adddate.surge.sh/

**Screenshots:**

### CODE REVIEW

**Comment by uday96** [comment]
@rishiraj824 @isuruAb @madhavrathi please review and suggest css for the date section.

File: src/components/ChatApp/ChatApp.css [hunk 1]
```diff
@@ -151,6 +151,18 @@ header{
     padding: 0;
 }
 
+.message-container.Date{
```
Comment (rishiraj824) [inline src/components/ChatApp/ChatApp.css]: the class message-container is containing a false ::after element which is creating a white arrow section. You can avoid using that class.

File: src/components/ChatApp/ChatApp.css [hunk 2]
```diff
@@ -151,6 +151,18 @@ header{
...
+    margin: 0 auto;
```
Comment (rishiraj824) [inline src/components/ChatApp/ChatApp.css]: Align the Date, not in center

**Comment by uday96** [comment]
@rishiraj824 made the requested changes.

**Comment by uday96** [comment]
@isuruAb @rishiraj824 aligned date to center also. check the demo link now

**Pull request merged**